wpilib/wpilib/analogtrigger.py

```diff
@@ -150,3 +158,7 @@ def createOutput(self, type):
         :returns: An AnalogTriggerOutput object.
         """
         return AnalogTriggerOutput(self, type)
+
+    def initSendable(self, builder):
+        if self.ownsAnalog:
+            self.analogInput.initSendable( )
```

Comment:
I feel there's something missing here.

Reply:
Looks like I tried to do ^Space, ended up doing just space